<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bilder verkleinern V1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>271</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Hochformat</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>227</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Max. Breite</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Max. Höhe</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboPortraitX">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboPortraitY">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>271</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Breitformat</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>228</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Max. Breite</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Max. Höhe</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboLandscapeX">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboLandscapeY">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Quadrat</string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Max. Breite/Höhe</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboSquareX">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Sonstiges</string>
     </property>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Qualität (%)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboQuality">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Zielverz.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboTarget">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>581</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSourceDir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>291</width>
       <height>26</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>/tmp/img</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFileSelect">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>21</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Auswahldialog</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConvert">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>270</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Alle Bilder aus dem Quellverzeichnis konvertieren und im Zielverzeichnis ablegen</string>
     </property>
     <property name="text">
      <string>Konvertieren</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>291</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Verzeichnis mit den Bildern (Quellen)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonStop">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>270</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Alle Bilder aus dem Quellverzeichnis konvertieren und im Zielverzeichnis ablegen</string>
     </property>
     <property name="text">
      <string>Beenden</string>
     </property>
    </widget>
    <zorder>buttonStop</zorder>
    <zorder>groupBox</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>groupBox_4</zorder>
    <zorder>listWidget</zorder>
    <zorder>comboSourceDir</zorder>
    <zorder>buttonFileSelect</zorder>
    <zorder>buttonConvert</zorder>
    <zorder>label_8</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConvert">
   <property name="text">
    <string>Konvertieren</string>
   </property>
   <property name="toolTip">
    <string>Alle Bilder in dem Verzeichnis gemäß den Vorgaben konvertieren. Ergebis im Unterverzeichnis &quot;mini&quot;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
